jobs/shield-agent/templates/bin/ctl

```diff
@@ -14,11 +14,19 @@ case $1 in
     echo "Attempting to set up SSH public keys"
     AUTHKEYS=/var/vcap/jobs/shield-agent/config/authorized_keys
     rm -f $AUTHKEYS
-    <% if_p("shield.agent.autoprovision") do |endpoint| %>curl -Lks "<%= endpoint %>/v1/meta/pubkey" -m 20 >> $AUTHKEYS || true<% end %>
-    <% if_p("shield.agent.daemon_public_key") do |key| %>cat >>$AUTHKEYS <<EOF
+    <%
+      if_p("autoprovision") do |endpoint|
+        if endpoint.to_s == "true"
+          endpoint = "http://#{link('shield-daemon').instances.first.address}:8080"
```

Comment:
Agreed. @geofffranks agreed to me merging nginx into shield-daemon job, so that one nice link will communicate the public https information; and we can close off :8080 again.